clusters/build-clusters/build11/multiarch-manager-operator/admin_01-operator.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
remove the `multiarch-manager-operator`, since we already added proper `nodeSelectors`